src/uct/tcp/tcp.h

```diff
@@ -284,10 +284,10 @@ typedef struct uct_tcp_ep_zcopy_tx {
  * TCP iface address flags
  */
 typedef enum uct_tcp_iface_addr_flags {
-    UCT_TCP_IFACE_ADDR_FLAG_LOCAL = UCS_BIT(0) /* Local address extension is
-                                                * included to iface address:
-                                                * uct_iface_ext_local_addr_t for
-                                                * loopback reachability */
+    UCT_TCP_IFACE_ADDR_FLAG_EXT = UCS_BIT(0) /* IFACE address is extended by
+                                              * additional information:
+                                              * uct_iface_ext_local_addr_t for
+                                              * loopback reachability */
```

Comment:
- pls add `@ref  uct_iface_ext_local_addr_t`, this way doxygen CI could help us to keep doc up to date even the type name is changed or removed
 - it's preferable to use doxygen format in docs comments (even for internal code)
 - minor: long comment better place in lines before
```suggestion
    /**
     * IFACE address is extended by additional information:
     * @ref uct_iface_ext_local_addr_t for loopback reachability
     */
    UCT_TCP_IFACE_ADDR_FLAG_EXT = UCS_BIT(0) 
```

Reply:
thanks
fixed